### DISCUSSION TITLE
How to build Optix on Windows

### DISCUSSION THREAD

**Discussion by Ash-LFC01** [vosen/ZLUDA#109] [Q&A]
I'm trying to do a GPU-based render on Autodesk 3ds Max using ZLUDA, and seemingly - it not only requires CUDA, but also the Optix library.

Here's how it appears normally, mentioning that it can't detect CUDA, naturally:
![image](https://github.com/vosen/ZLUDA/assets/100218808/2df13898-934c-4c75-bcec-d8baaff53f10)

And how it appears after I launch with ZLUDA, where it does detect CUDA - which is nice, but I still can't render without Optix:
![image](https://github.com/vosen/ZLUDA/assets/100218808/52fe9b4b-4f2a-4d5a-872d-1fa34eb4c5ed)

The documentation only provided the steps to build ZLUDA-OptiX on Linux, is it possible on Windows too?

**Comment by ginolig** [comment]
Maybe it can be cross-compiled? I know Rust allows cross compilation of apps but I'm not sure if also for libraries
